.ruby-version

```diff
@@ -1 +1 @@
-jruby-9.2.11.1
+2.6.6
```

Comment:
Do we want ruby version to be jruby?

Reply:
yessss! good catch :) thanks! will fix it